File: BE-Cinema/src/main/java/com/datn/cinema/dto/MovieTicketDTO.java
package com.datn.cinema.dto;

public class MovieTicketDTO {
    private Integer movieId;
    private Integer showTime;
    private String showDate;
    private Integer roomId;
    private Integer projectionId;


    public MovieTicketDTO(Integer movieId, Integer showTime, String showDate, Integer roomId, Integer projectionId) {
        this.movieId = movieId;
        this.showTime = showTime;
        this.showDate = showDate;
        this.roomId = roomId;
        this.projectionId = projectionId;
    }

    public Integer getMovieId() {
        return movieId;
    }

    public void setMovieId(Integer movieId) {
        this.movieId = movieId;
    }

    public Integer getShowTime() {
        return showTime;
    }

    public void setShowTime(Integer showTime) {
        this.showTime = showTime;
    }

    public String getShowDate() {
        return showDate;
    }

    public void setShowDate(String showDate) {
        this.showDate = showDate;
    }

    public Integer getRoomId() {
        return roomId;
    }

    public void setRoomId(Integer roomId) {
        this.roomId = roomId;
    }

    public Integer getProjectionId() {
        return projectionId;
    }

    public void setProjectionId(Integer projectionId) {
        this.projectionId = projectionId;
    }
}

File: FE-Cinema/src/app/entity/movieTicketDTO.ts
export class MovieTicketDTO {
  private _movieId: number;
  private _showTimeId: number;
  private _showDate: string;
  private _roomId: number;
  private _projectionTypeId: number;


  constructor(movieId: number, showTimeId: number, showDate: string, roomId: number, projectionTypeId: number) {
    this._movieId = movieId;
    this._showTimeId = showTimeId;
    this._showDate = showDate;
    this._roomId = roomId;
    this._projectionTypeId = projectionTypeId;
  }

  get movieId(): number {
    return this._movieId;
  }

  set movieId(value: number) {
    this._movieId = value;
  }

  get showTimeId(): number {
    return this._showTimeId;
  }

  set showTimeId(value: number) {
    this._showTimeId = value;
  }

  get showDate(): string {
    return this._showDate;
  }

  set showDate(value: string) {
    this._showDate = value;
  }

  get roomId(): number {
    return this._roomId;
  }

  set roomId(value: number) {
    this._roomId = value;
  }

  get projectionTypeId(): number {
    return this._projectionTypeId;
  }

  set projectionTypeId(value: number) {
    this._projectionTypeId = value;
  }
}
